pytest_fixtures/component/activationkey.py

```diff
@@ -6,8 +6,14 @@
 
 
 @pytest.fixture(scope='module')
-def module_activation_key(module_org, module_target_sat):
-    return module_target_sat.api.ActivationKey(organization=module_org).create()
+def module_activation_key(module_entitlement_manifest_org, module_target_sat):
```

Comment:
There's a fixture with same name and similar content in here: https://github.com/SatelliteQE/robottelo/blob/5405d12dfb0e66ec879707b1af85c11bb9afc901/tests/foreman/ui/test_host.py#L101
Can we have just one fixture for both?

Reply:
@rmynar The one in test_host.py has more steps which is specific for the module.
For `SCA mode manifest activation key`, I don't see it being used at many places. We can add one if needed.

Reply:
I updated the fixture to use sca enabled manifest.